### ISSUE TITLE
Regression after merge #fix-127-trampled-units

### ISSUE THREAD

**Issue by maksimru** [jordanbrauer/unit-converter#139]
Issue wasn't appear before merge of pull #132 

Can be reproduced with following test case

```php
    /**
     * @test
     */
    public function assertAsUnitConversationWorkForSameUnits()
    {
        $value = new Kilometre(1);
        $this->assertEquals(1, $value->as(new Kilometre()));
    }
```

After pull above it started to throw exception **UnitConverter\Exception\BadRegistry : Unit 'km' is already registered. Please use a different notation or remove the conflicting unit**

**Comment by jordanbrauer** [comment]
Thanks for the report @maksimru!

Surprising that it caused a regression – I'll have to check the tests again to see what it is that I missed.

I can whip up a fix sometime this weekend. Apologies for any inconvenience in the mean time!

**Comment by jordanbrauer** [comment]
@maksimru bug is fixed and released in latest version `0.8.3` 🎊 

```
➜  Successfully bumped version to v0.8.3
```
